### PR TITLE
docs(_SETUP.md): add update example use

### PR DESCRIPTION
**Pull request by shaodahong**
**What kind of change does this PR introduce?**
add `_SETUP.md` update example use

**Did you add tests for your changes?**
Yes

**Does this PR introduce a breaking change?**
No

**What needs to be documented once your changes are merged?**
Nothing

### CODE REVIEW

File: _SETUP.md [hunk 1]
```diff
@@ -87,3 +87,16 @@ or incremental (in watch mode)
...
 yarn type-lint --watch
 ```
+
+### To update all examples use
+
+```bash
+yarn build:examples
+```
+
+### To update some example use
```
Comment (rishabh3112) [inline _SETUP.md]:
```suggestion
### To update a specific example use
```

File: _SETUP.md [hunk 2]
```diff
@@ -87,3 +87,16 @@ or incremental (in watch mode)
...
+
+```bash
+cd examples/(some example dir)
```
Comment (rishabh3112) [inline _SETUP.md]:
```suggestion
cd examples/<path to example>
```

**Comment by shaodahong** [comment]
I changed, thank you

**Comment by sokra** [comment]
Thanks

**Pull request merged**